docs/hosting/configuration/environment-variables/nodes.md

```diff
@@ -25,3 +25,4 @@ This page lists the environment variables configuration options for managing nod
 | `NODES_ERROR_TRIGGER_TYPE` | String | `n8n-nodes-base.errorTrigger` | Specify which node type to use as Error Trigger. |
 | `N8N_CUSTOM_EXTENSIONS` | String | - | Specify the path to directories containing your custom nodes. |
 | `N8N_COMMUNITY_PACKAGES_ENABLED` | Boolean | `true` | Enables (true) or disables (false) community nodes. |
+| `N8N_COMMUNITY_PACKAGES_REGISTRY` | String | `https://registry.npmjs.org` | NPM registry URL to pull community packages from. |
```

Comment:
@Joffcom we need to add a note here saying that this will only work for enterprise or startup license. Any preference on how and where to add that note?

Reply:
For multi main we just add `(license required)` I would do the same here